atlas/templates/cmd/server/main.go.gotmpl

```diff
@@ -162,6 +165,20 @@ func forwardResponseOption(ctx context.Context, w http.ResponseWriter, resp prot
 	return nil
 }
 {{- end }}
+
+{{ if .WithGateway -}}
+func whitelistStringsHeaderMatcher(headerNames ...string) func(string) (string, bool) {
```

Comment:
I strongly recommend move this util function into toolkit instead of generating new one for every application

Reply:
Good idea.